tools/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/cfergeau/win32-background-launcher
+module github.com/crc-org/win32-background-launcher
```

Comment:
I'd name this `github.com/crc-org/win32-background-launcher/tools` but I don't know if this is important!

Reply:
updated in the original commit, this should've been part of it so fixed that now in https://github.com/crc-org/win32-background-launcher/pull/1/commits/3be44d1ac35d7410dddb702cf55b31d8479cd8f8